homeassistant/components/fritzbox/__init__.py

```diff
@@ -44,14 +43,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
         CONF_CONNECTIONS: fritz,
     }
 
-    try:
-        await hass.async_add_executor_job(fritz.update_templates)
-    except ParseError:
-        LOGGER.debug("Disable smarthome templates")
-        has_templates = False
-    else:
-        LOGGER.debug("Enable smarthome templates")
-        has_templates = True
+    has_templates = await hass.async_add_executor_job(fritz.has_templates)
+    LOGGER.debug("enable smarthome templates: %s", has_templates)
```

Comment:
Nit: Capitallize **E** nable

Reply:
this is an debug message and does not need to be capitalized